### PR TITLE
Optionally specify a separate hostUri for web socket connection

### PR DESCRIPTION
**Pull request by computmaxer**
@ekweible

### CODE REVIEW

File: lib/src/transport/transport.dart [hunk 1]
```diff
@@ -13,10 +13,11 @@ class Transport {
 
   WebSocketTransport _webSocket;
 
-  Transport(Session session)
+  Transport(Session session, {Uri websocketHost})
```
Comment (ekweible) [inline lib/src/transport/transport.dart]: `webSocketHost`

Comment (computmaxer) [inline lib/src/transport/transport.dart]: yikes

Comment (computmaxer) [inline lib/src/transport/transport.dart]: Fixed

**Comment by ekweible** [comment]
+1

**Pull request merged**